docs/source/model_design/vivarium_features/other_causes_ylds/index.rst

```diff
@@ -210,34 +210,95 @@ background morbidity, including:
+  - Does not adjust for comorbidity due to unmodeled causes, resulting in overestimation of YLDs due to modeled causes relative to GBD estimates
 
 - Underestimation of total YLDs
 
   - Only observe a subset of total YLDs within our simulation. While we can calculate YLDs averted between scenarios, we cannot accurately calculate percent reduction in all-cause YLDs or DALYs relative to baseline because we do not model YLDs due to all causes at baseline.
 
-  - Causes us to overestimate impact of a death averted in our simulation. An averted death in the alterantive relative to baseline scenario result in some number of YLLs averted, but really this person should then start accruing YLDs overtime, which will decrease the number of DALYs averted relative to baseline. 
+  - Causes us to overestimate impact of a death averted in our simulation. An averted death in the alternative relative to baseline scenario result in some number of YLLs averted, but really this person should then start accruing YLDs overtime, which will decrease the number of DALYs averted relative to baseline. 
 
 Proposal
 --------------
 
 To address these issues, we propose to update the default vivarium behavior to 
 model background morbidity in a similar manner to the default behavior to model 
-background mortality. 
+background mortality. Additionally, we propose the incorporation of cause-specific 
+COMO-adjusted YLDs into vivarium observers.
+
+1. Background morbidity
++++++++++++++++++++++++
+
+In order to model YLDs due to non-modeled background causes, we must estimate a 
+"cause-deleted" disability weight that represents the disability weight for all
+GBD causes except for those explicitly modeled in a given vivarium simulation. 
+This "cause-deleted" disability weight should be adjusted for comorbidity of all
+individual causes of disability included in background morbidity, but NOT adjusted
+for comorbidity with modeled causes (which instead will be performed within 
+the simulation).
+
+The ideal approach to estimating this cause-deleted COMO adjusted disability weight
+due to unmodeled causes would be to perform the COMO adjustment as is done in the 
+central computation process while excluding modeled sequelae. This should be done
+at the location/age/sex/year-specific level. Integrating the central computation
+code into vivarium processes would be ideal so that we will replicate all of the
+exceptions and maintain any regular updates to their process.
+
+However, until this is able to be achieved, an interim solution will be to estimate
+the DW due to background morbidity as the difference between the all-cause YLD rate 
+and the sum of the YLD rate due to all modeled causes of disability at the 
+location/age/sex/year-specific level:
+
+.. math::
+
+  DW_\text{background} = \text{YLD rate}_\text{c294} - \sum_{c=1}^n \text{YLD rate}_c
```

Comment:
This gives me an idea, but I haven't fully thought it through. What if we defined:

*YLD_modeled = sum of YLD rates for modeled causes*

*YLD_background = YLD_c294 - YLD_modeled*

And then solved this system of two equations with two unknowns:

*YLD_c294 = 1 - (1 - DW_background) * (1 - DW_modeled)*

*DW_background / DW_modeled = YLD_background / YLD_modeled*

At first glance that seems like it would adjust for the issue you note below about modeled YLDs being a large proportion of YLDs, assuming independence. I'm not super confident about the last equation but it seems intuitively valid.

Reply:
This is interesting! I think there is potentially something here, but I don't think it works as written due to combining population-level and individual-level equations.

The relationship between YLDs and DWs depends on the prevalence of the condition at hand. We've been bypassing this for background mortality because we are assuming the prevalence of this condition to be 1, so in this "special" case _YLDs_background = DW_background_, but (assuming there is just one modeled cause for simplicity) _YLDs_modeled = DW_modeled * prevalence_modeled_. 

I think we might be able to do the following...

_YLDs_c294 = prevalence_modeled * (1 - (1 - DW_background) * (1 - DW_modeled)) + (1 - prevalence_modeled) * (DW_background)

DW_background / (DW_modeled * prevalence_modeled) = YLD_background / YLD_modeled_

I _think_ this holds for a single cause of disability, but I think it gets more complicated if we are modelling multiple causes... but I believe that under the assumption of independence all causes should have approximately the same "deflation" factor in their adjusted disability weight to their unadjusted disability weight from the COMO adjustment, so I think that the second equation should hold when we consider the prevalence.

Any thoughts?

Reply:
I'll try to run some test calculations/adjustments in the meantime to confirm the suspicion!

Reply:
Oh wait... actually I think this doesn't really work because this equation _YLDs_modeled = DW_modeled * prevalence_modeled_ that I said was correct is not actually correct. It's really  _YLDs_modeled = ADW_modeled * prevalence_modeled_, where ADW is the adjusted DW, which does not equal the DW.

BUT, DW_background is the only undefined term in this equation, so we might be able to solve it directly and go with that?

_YLDs_c294 = prevalence_modeled * (1 - (1 - DW_background) * (1 - DW_modeled)) + (1 - prevalence_modeled) * (DW_background)_

Then we only need to think through how to make this scale for more than a couple causes at a time...

Reply:
Sorry -- I don't think I was completely clear here. For DW_modeled I don't propose using an actual DW from GBD. Instead, it's just one of the two unknowns we are solving for. We wouldn't actually do anything with the solved value for DW_modeled.

I was thinking about it as if both "background" and "modeled" were conditions with prevalence 100%. I believe that simplifies the issue of multiple modeled conditions as well?